Readme.md

```diff
@@ -33,4 +33,30 @@ for event in in_events {
 writer.finish()?;
 ```
 
+## Async API
+
+By default this crate enables the `sync` feature which exposes a sync API. You
+can however switch to using a `tokio`-backed async API by disabling the `sync`
+feature and enabling the `tokio` feature.
+
+Either run the following in the root of your crate:
+
+```sh
+cargo add hepmc2 --no-default-features -F tokio
+```
+
+or make sure a line like the following is present in your `Cargo.toml`:
+
+```toml
+hepmc2 = { version = "0.6.0", default-features = false, features = ["tokio"] }
+```
+
+The async API is exactly the same as the sync one but IO operations will return
+futures that you will, as usual, need to call `await` on. For examples, generate
+the async API documentation in the root of this project:
+
+```sh
+cargo doc --open --no-default-features -F tokio
+```
+
```

Comment:
I'd like this to also be in the `lib.rs` docs, see below.

Reply:
Done